types/nightwatch/index.d.ts

```diff
@@ -6572,13 +6740,15 @@ export interface WebDriverProtocolUserActions {
      * this.demoTest = function (browser) {
      *   browser.moveTo(null, 110, 100);
      * };
+     *
+     * @see https://nightwatchjs.org/api/moveTo.html#apimethod-container
      */
     moveTo(
-        element: string | null,
+        id: string | null,
         xoffset: number,
         yoffset: number,
-        callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<void>) => void,
-    ): this;
+        callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<null>) => void,
+    ): Awaitable<this, null>;
```

Comment:
This will have three overloads after the recent changes: https://github.com/nightwatchjs/nightwatch/commit/f9822193e3b4672fb415cde2c3e08db5397b7590

First with just `elementId`, second with just the offsets and the third with both `elementId` and offsets.